hibike/hibike_message.py

```diff
@@ -437,10 +448,13 @@ def blocking_read_generator(serial_conn, stop_event=None):
                 packets_buffer = packets_buffer[new_packet:]
             # Otherwise, there might be more incoming bytes for the current packet,
             # so we do a blocking read and try again
-            else:
+            elif not stop_event.is_set():
```

Comment:
Actually, this particular check is not necessary anyways, since reads are nonblocking. I will remove that.